packages/amplify-graphql-model-transformer/rds-patching-lambda/src/index.ts

```diff
@@ -0,0 +1,61 @@
+import { LambdaClient, UpdateFunctionConfigurationCommand, UpdateFunctionConfigurationCommandOutput } from "@aws-sdk/client-lambda";
+
+const snsEventSource = 'aws:sns';
+
+type LayerConfig = {
+  layerArn: string;
+  region: string;
+};
+
+const getLayerConfig = (event: any): LayerConfig => {
```

Comment:
There should be an `SNSEvent` type we can use here.

Reply:
I intentionally avoided it so that the lambda can be invoked without the full event object. Still thought about creating minimum type definition for an event but the 'MessageAttributes' aren't strongly typed. But we can still do better than 'any', I will take of it in a follow up PR.